test/e2e/specs/specs-playwright/wp-support__search-spec.js

```diff
@@ -0,0 +1,183 @@
+/**
+ * External dependencies
+ */
+import assert from 'assert';
+import {
+	DataHelper,
+	LoginFlow,
+	SidebarComponent,
+	MyHomePage,
+	SettingsPage,
+	SupportComponent,
+} from '@automattic/calypso-e2e';
+
+describe( DataHelper.createSuiteTitle( 'Support' ), function () {
+	describe( 'Search for a support topic then close popover', function () {
+		let supportComponent;
+
+		it( 'Log in', async function () {
+			const loginFlow = new LoginFlow( this.page );
+			await loginFlow.login();
+		} );
+
+		it( 'Open Settings page', async function () {
+			await MyHomePage.Expect( this.page );
+			const sidebarComponent = await SidebarComponent.Expect( this.page );
+			await sidebarComponent.clickMenuItem( 'Settings' );
+		} );
+
+		it( 'Open support popover', async function () {
+			await SettingsPage.Expect( this.page );
+			supportComponent = await SupportComponent.Expect( this.page );
+			await supportComponent.clickSupportButton();
+		} );
+
+		it( 'Displays default entries', async function () {
+			const results = await supportComponent.getOverallResultsCount();
+			// Default entries varies from 5 on TeamCity to 6 when run locally.
```

Comment:
For more context, this number is dynamic as it depends on whether the user has currently cancellable purchases in progress. If they do, then the additional sixth `Manage purchases` item is added. This has been a source of flaky behavior (p1623054412080100-slack-C01QG4Y91RR). Generally the expected number should be exactly 5, but because some other specs running in parallel on the same site were bleeding into this one, from time to time the 6th one appeared.

Reply:
cc @scinos

Reply:
This should probably be kept in mind that when migrating specs that involve any kind of purchasing, they should all be running against an isolated, dynamically created page. Also, do you think the 6th element I mentioned above should be a subject for a separate test case? @worldomonation @bsessions85

Reply:
You're right, i checked between two sites I have - one with purchases and one without, and the default entries do change as you stated!

I think this should be codified in a test and for us to set up additional e2e accounts:
- has purchases
- has no purchases

We already have a bunch of e2e accounts that restore to a known good state at certain time of the day. For these two scenarios, we don't need to actively isolate the tests in my opinion as we are not testing something like the `cart`. Who would we talk to get these accounts managed like this? @WunderBart

Reply:
Not sure I understand, sorry. Do you mean we need two **new** e2e accounts/pages just for this suite? As in one with the pre-initiated purchase process and other without?

Reply:
> Do you mean we need two new e2e accounts/pages just for this suite? 

Sure, if that's what it takes. 

What I am proposing as the goal is to set up two accounts, one with a purchase and one without. That's the only condition that separates the default search results of 6 and 5 entries respectively.

The purchase process is unnecessary as far as I can tell, so there is no risk of tests tripping over each other.